scipy/special/_precompute/cosine_cdf.py

```diff
@@ -1,5 +1,5 @@
 
-import mpmath
+import mpmath # type: ignore[import]
```

Comment:
```suggestion
import mpmath  # type: ignore[import]
```

Two spaces according to flake8 E261 convention. I see a warning generated but don't see failures because of this. Does scipy still use/prefer flake8 standards?